src/components/ClaimRow/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum ClaimStatus {
+  Available = 'Available',
+  NotYetAvailable = 'Not Yet Available',
```

Comment:
Is "Pending" or "CoolingDown" a better name for this?

Reply:
Agree, went with "pending" which generalises outside of rFOX 👍